seed/static/seed/js/controllers/mapping_controller.js

```diff
@@ -64,112 +64,123 @@ angular.module('BE.seed.controller.mapping', [])
       Notification,
       organization_payload,
       naturalSort,
-      COLUMN_MAPPING_PRESET_TYPE_NORMAL,
-      COLUMN_MAPPING_PRESET_TYPE_BUILDINGSYNC_DEFAULT,
-      COLUMN_MAPPING_PRESET_TYPE_BUILDINGSYNC_CUSTOM
+      COLUMN_MAPPING_PROFILE_TYPE_NORMAL,
+      COLUMN_MAPPING_PROFILE_TYPE_BUILDINGSYNC_DEFAULT,
+      COLUMN_MAPPING_PROFILE_TYPE_BUILDINGSYNC_CUSTOM
     ) {
-      $scope.presets = [
+      $scope.profiles = [
         {id: 0, mappings: [], name: '<None selected>'}
-      ].concat(column_mapping_presets_payload);
+      ].concat(column_mapping_profiles_payload);
 
       // $scope.selected_preset = $scope.applied_preset = $scope.mock_presets[0];
-      $scope.dropdown_selected_preset = $scope.current_preset = $scope.presets[0] || {};
+      $scope.dropdown_selected_profile = $scope.current_profile = $scope.profiles[0] || {};
       $scope.organization = organization_payload.organization;
 
       // Track changes to help prevent losing changes when data could be lost
-      $scope.preset_change_possible = false;
+      $scope.profile_change_possible = false;
 
-      $scope.flag_preset_change = function () {
-        $scope.preset_change_possible = true;
+      $scope.flag_profile_change = function () {
+        $scope.profile_change_possible = true;
       };
 
       $scope.flag_mappings_change = function () {
         $scope.mappings_change_possible = true;
       };
 
-      $scope.is_buildingsync_and_preset_not_ok = function () {
+      $scope.is_buildingsync_and_profile_not_ok = function () {
         if (!$scope.mappingBuildingSync) {
           return false;
         }
-        // BuildingSync requires a saved preset to be applied
-        return $scope.current_preset.id === 0
-          || $scope.preset_change_possible
+        // BuildingSync requires a saved profile to be applied
+        return $scope.current_profile.id === 0
+          || $scope.profile_change_possible
           || $scope.mappings_change_possible;
       };
 
-      $scope.apply_preset = function () {
+      var analyze_chosen_inventory_types = function () {
```

Comment:
I think I did this... I'm a bit worried now that I botched the merge conflict using the GH UI to do it 🤦 .

This function was removed here: https://github.com/SEED-platform/seed/pull/2372/files

Thinking we revert locally, update with develop, then force push up?